testing/armagetronad/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+# Contributor: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=armagetronad
+pkgver=0.2.8.3.4
+pkgrel=0
+pkgdesc="A Tron Clone in 3D."
+url="http://armagetronad.net/"
+arch="all"
+license="GPL-1.0-only"
+makedepends="sdl_image-dev sdl_mixer-dev ftgl-dev libxml2-dev"
+subpackages="$pkgname-doc $pkgname-openrc"
+options="!check" #the tests are deactivated because the make check has not been declared
+source="https://launchpad.net/armagetronad/${pkgver%.*.*}/$pkgver/+download/$pkgname-$pkgver.src.tar.gz"
+
+prepare(){
+    default_prepare
+    update_config_guess
+}
+
+build() {
+  ./configure \
+      --prefix=/usr \
+      --disable-useradd \
+      --enable-dirty \
+      --enable-music \
+      --enable-armathentication \
+      --enable-master \
+      --enable-krawall \
+      --bindir=/usr/bin \
+      --sbindir=/usr/sbin \
+      --libexecdir=/usr/libexec \
+      --datadir=/usr/share \
+      --sysconfdir=/usr/share \
+      --sharedstatedir=/usr/share \
+      --localstatedir=/var \
+      --libdir=/usr/share \
+      --includedir=/usr/share \
+      --docdir=/usr/share/doc/armagetronad
+  make
+}
+
+
+package() {
+    make DESTDIR="$pkgdir" install
```

Comment:
Seems like there's a racecondition in the build system. Could you please change this to `make -j1 DESTDIR="$pkgdir" install` to see if that fixes it?

Reply:
ok

Reply:
Seems to work, thanks